README.md

```diff
@@ -5,10 +5,12 @@ models encoded in [ONNX](https://github.com/onnx/onnx).
 
 ## Quick Start
 
+Agate supports Scala 2.11, 2.12, and 2.13.
+
 To include Agate in your SBT project, use the following snippet:
 
 ```scala
-libraryDependencies += "com.stripe" %% "agate-core" % "0.1.0"
+libraryDependencies += "com.stripe" %% "agate-core" % "0.0.10"
```

Comment:
this is wrong. I published 0.1.0. We seem to have rolled back some.

Reply:
Oh, I templated it wrong. Sorry, will fix!

Reply:
wait, did you update the README.md or just the template source?